iPetCare/Persistence/DataContext.cs

```diff
@@ -1,11 +1,33 @@
 ﻿using Domain.Models;
 using Microsoft.AspNetCore.Identity.EntityFrameworkCore;
 using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Storage.ValueConversion;
 
 namespace Persistence
 {
     public class DataContext : IdentityDbContext<ApplicationUser>
     {
+        public DbSet<Examination> Examinations { get; set; }
+        public DbSet<ExaminationParameter> ExaminationParameters { get; set; }
+        public DbSet<ExaminationParameterValue> ExaminationParameterValues { get; set; }
+        public DbSet<ExaminationType> ExaminationTypes { get; set; }
+        public DbSet<ImportantDate> ImportantDates { get; set; }
+        public DbSet<Institution> Institutions { get; set; }
+        public DbSet<Note> Notes { get; set; }
+        public DbSet<Owner> Owners { get; set; }
+        public DbSet<Pet> Pets { get; set; }
+        public DbSet<Race> Races { get; set; }
+        public DbSet<Request> Requests { get; set; }
+        public DbSet<Species> Species { get; set; }
+        public DbSet<Vet> Vets { get; set; }
+
+        // association tables
+        public DbSet<ImportantDatePet> ImportantDatePets { get; set; }
+        public DbSet<InstitutionVet> InstitutionVets { get; set; }
+        public DbSet<OwnerPet> OwnerPets { get; set; }
+        public DbSet<VetPet> VetPets { get; set; }
+
+
         public DataContext(DbContextOptions options) : base(options)
```

Comment:
Może warto tutaj zdefiniować zachowania podczas usuwania (tj kaskadowość)?

Reply:
O których encjach konkretnie mówisz? Tzn. w niektórych EF nie wygenerował usuwania kaskadowego, tak?

Reply:
Chodzi o to, żeby zdefiniować coś takiego:
https://entityframeworkcore.com/saving-data-cascade-delete

Na przykład jak usuwamy AspNetIdentityUser to chcemy, żeby usunął się Vet/Owner itp